olm/olm.sh

```diff
@@ -499,6 +499,21 @@ waitCheServerDeploy() {
   fi
 }
 
+getCatalogSourcePod() {
```

Comment:
I need this function to wait for catalog source to be created in cluster. Will be neded for che oeprator update tests in ocp 4.7, 4.8